src/components/App.tsx

```diff
@@ -45,7 +45,7 @@ import { Button } from "./Button";
 import * as ReactModal from "react-modal";
 import { NewFileDialog } from "./NewFileDialog";
 import { EditFileDialog } from "./EditFileDialog";
-import { Toast } from "./Toast";
+import { ToastContainer } from "./Toasts";
```

Comment:
The name of the file seems unnatural `Toasts` -> `ToastContainer` would be more appropriate?